t3f/ops.py

```diff
@@ -1191,3 +1192,44 @@ def renormalize_tt_cores(tt, epsilon=1e-8):
 
       return TensorTrainBatch(new_cores)
 
+
+def ones_like(tt):
+  """Constructs t3f.ones with the shape of `tt`.
```

Comment:
there is no t3f.ones.
Returns TT-object with the same shape as the input `tt` but with all elements equal to 1. (if this fits to line limit)